libs/utils/utils.js

```diff
@@ -853,6 +855,31 @@ async function checkForPageMods() {
     persManifests = persManifests.concat(getPromoManifests(promoMd));
   }
 
+  if (dynamicNavEnabled) {
+    const gns = getMetadata('gnav-source');
```

Comment:
we should use descriptive names

Reply:
Ah, great catch, thank you, I meant to change the name.